### PR TITLE
[4.3] Guided Tours Caption [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
The key used for the caption was incorrect resulting in an untranslated string.

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by richard67** [comment]
@brianteeman The diff on GitHub shows a lot more changes than just the language string key.

**Comment by brianteeman** [comment]
@richard67 stupid ide - fixing

**Comment by Quy** [comment]
There isn't a language string for it, but this should  be `List of Steps`.

**Comment by brianteeman** [comment]
> There isn't a language string for it, but this should be `List of Steps`.

good spot - updated the pr now

File: administrator/language/en-GB/com_guidedtours.ini [hunk 1]
```diff
@@ -65,6 +65,7 @@ COM_GUIDEDTOURS_STEP_TYPE="Type"
 COM_GUIDEDTOURS_STEP_TYPE_LABEL="Type"
 COM_GUIDEDTOURS_STEP_TYPE_DESC="Select <em>Next</em> for a walk-through, <em>Redirect</em> to redirect to another page, or <em>Interactive</em> to allow user interaction."
 COM_GUIDEDTOURS_STEPS="Steps"
+COM_GUIDEDTOURS_STEPS_TABLE_CAPTION="List of Steps"
```
Comment (Quy) [inline administrator/language/en-GB/com_guidedtours.ini]: Please alpha sort.

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on c7343396f87c50adc9842ba335801757e1ad1f56<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40214">issues.joomla.org/tracker/joomla-cms/40214</a>.</sub>

**Comment by obuisard** [comment]
Thank you Brian @brianteeman !

**Pull request merged**